src/test/resources/com/spotify/github/v3/githubapp/installation.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
minor: it's perhaps good to not use "real world" response examples as fixtures so that it's obvious from the tests that we're not calling an external endpoint. E.g login: "github-test"

Reply:
all of the fixtures across the whole client are taken from the examples in the github API docs :) 
https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-a-repository-installation-for-the-authenticated-app